src/jwplayer/utils/iasTracker.ts

```diff
@@ -0,0 +1,25 @@
+import { scriptLoader } from 'jwplayer/utils/scriptLoader';
+
+const scriptUrl = '//static.adsafeprotected.com/vans-adapter-google-ima.js';
+class IasTracker {
+	private scriptPromise: Promise<Event>;
```

Comment:
I'd suggest adding the `undefined` type to this property since it doesn't have an initializer and isn't assigned within the constructor.